src/attributes/views/AttributeCreate/AttributeCreate.tsx

```diff
@@ -145,10 +145,12 @@ const AttributeDetails: React.FC<AttributeDetailsProps> = ({ params }) => {
     setValues(move(values[oldIndex], values, areValuesEqual, newIndex));
 
   const handleCreate = async (data: AttributePageFormData) => {
-    const input =
-      data.inputType === AttributeInputTypeEnum.FILE
-        ? getFileAttributeData(data, values)
-        : getSimpleAttributeData(data, values);
+    const input = [
+      AttributeInputTypeEnum.FILE,
```

Comment:
Seems like this array is actually in some different places, maybe it'd be worth to make a const of it and export it?

Reply:
it seems to be good idea